package.json

```diff
@@ -9,8 +9,8 @@
     "lodash.clonedeep": "^4.5.0",
     "lodash.truncate": "^4.4.2",
     "slice-ansi": "^4.0.0",
-    "string-width": "^4.2.0",
-    "strip-ansi": "^6.0.0"
+    "string-width": "^4.2.3",
```

Comment:
No chance you could possible switch to switch to ESM also and update to 5.x?
:)

Reply:
It will a breaking change for our users since those versions drop Node v10. 
Is there any issues for your project with this patch change?

Reply:
Nope, i just wish more ppl jump on the esm wagon 
All sindre 1000 repos gets converted to esm only
Babel 8 looks like it's going to be esm only
And node-fetch is also esm only
And node 10 don't get any more support

Reply:
Yep I think it's good point. ESM is going to become a new standard module loader. Hope that the package will adapt the trend soon.